.github/workflows/release.yaml

```
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v**'
```

```suggestion
on:
  release:
    types: [created]
```

This is triggered by tags, etc too.

---

Are you sure that event makes sense?

I thought it's for actual releases - e.g. https://github.com/submariner-io/submariner-operator/releases

In such a case it will *only* trigger for tags but never for merge to the master branch..

Please correct me if I'm understanding this wrong

---

I think you're right. I forgot of master